<ui version="4.0" >
 <author>Christian Schenk</author>
 <class>ConnectionSettingsDialog</class>
 <widget class="QDialog" name="ConnectionSettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Connection Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="grpUseProxy" >
     <property name="title" >
      <string>&amp;Use a proxy server</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&amp;Address:</string>
          </property>
          <property name="buddy" >
           <cstring>leAddress</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leAddress" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>15</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&amp;Port:</string>
          </property>
          <property name="buddy" >
           <cstring>lePort</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePort" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAuthRequired" >
        <property name="text" >
         <string>Au&amp;thentication required</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leAddress</tabstop>
  <tabstop>lePort</tabstop>
  <tabstop>chkAuthRequired</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>168</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel" >
     <x>168</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>168</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel" >
     <x>168</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
